wemake_python_styleguide/violations/best_practices.py

```diff
@@ -2572,3 +2574,33 @@ def my_function(): ...
 
     error_template = 'Found new-styled decorator'
     code = 466
+
+
+@final
+class GettingFirstElementByUnpackingViolation(ASTViolation):
+    """
+    Forbid getting first element using unpacking.
+
+    Reasoning:
+        Performance. Prefixing unused variables with underscore is nothing
+        more than convention, Python still creates these variables.
+        So, unpacking above makes a new unused list which is slow.
+
+    Solution:
+        Use `collection[0]` or `next(iter(collection))`
+
+    Example::
+
+        # Correct:
+        first = some_collection[0]
+        first = next(iter(collection))
+
+        # Wrong:
+        first, *_rest = some_collection
+
+    .. versionadded:: 0.16.0
+
+    """
+
+    error_template = 'Found incorrect unpacking target'
```

Comment:
This message is identical to `WrongUnpackingViolation.error_template`, let's change it to something more related to this violation, e.g.:

```suggestion
    error_template = 'Found wrong retrieving of the first element'
```

Reply:
Thanks, I forget to fix this after copy-paste. Done.